tools/version-tracker/pkg/constants/constants.go

```diff
@@ -6,41 +6,74 @@ import (
-	HeadRepoOwnerEnvvar                      = "HEAD_REPO_OWNER"
-	GitHubTokenEnvvar                        = "GITHUB_TOKEN"
-	CommitAuthorNameEnvvar                   = "COMMIT_AUTHOR_NAME"
-	CommitAuthorEmailEnvvar                  = "COMMIT_AUTHOR_EMAIL"
-	DefaultCommitAuthorName                  = "EKS Distro PR Bot"
-	DefaultCommitAuthorEmail                 = "aws-model-rocket-bots+eksdistroprbot@amazon.com"
-	BuildToolingRepoName                     = "eks-anywhere-build-tooling"
-	DefaultBaseRepoOwner                     = "aws"
-	BuildToolingRepoURL                      = "https://github.com/%s/eks-anywhere-build-tooling"
-	ReadmeFile                               = "README.md"
-	ReadmeUpdateScriptFile                   = "build/lib/readme_check.sh"
-	LicenseBoilerplateFile                   = "hack/boilerplate.yq.txt"
-	SkippedProjectsFile                      = "SKIPPED_PROJECTS"
-	UpstreamProjectsTrackerFile              = "UPSTREAM_PROJECTS.yaml"
-	GitTagFile                               = "GIT_TAG"
-	GoVersionFile                            = "GOLANG_VERSION"
-	ChecksumsFile                            = "CHECKSUMS"
-	AttributionsFilePattern                  = "*ATTRIBUTION.txt"
-	PatchesDirectory                         = "patches"
-	BottlerocketReleasesFile                 = "BOTTLEROCKET_RELEASES"
-	BottlerocketAdminContainerMetadataFile   = "BOTTLEROCKET_ADMIN_CONTAINER_METADATA"
-	BottlerocketControlContainerMetadataFile = "BOTTLEROCKET_CONTROL_CONTAINER_METADATA"
-	GithubPerPage                            = 100
-	datetimeFormat                           = "%Y-%m-%dT%H:%M:%SZ"
-	MainBranchName                           = "main"
-	BaseRepoHeadRevision                     = "refs/remotes/origin/main"
-	PullRequestBody                          = `This PR bumps %[1]s/%[2]s to the latest Git revision, along with other updates such as Go version, checksums and attribution files.
+	BaseRepoOwnerEnvvar                     = "BASE_REPO_OWNER"
+	HeadRepoOwnerEnvvar                     = "HEAD_REPO_OWNER"
+	GitHubTokenEnvvar                       = "GITHUB_TOKEN"
+	CommitAuthorNameEnvvar                  = "COMMIT_AUTHOR_NAME"
+	CommitAuthorEmailEnvvar                 = "COMMIT_AUTHOR_EMAIL"
+	DefaultCommitAuthorName                 = "EKS Distro PR Bot"
+	DefaultCommitAuthorEmail                = "aws-model-rocket-bots+eksdistroprbot@amazon.com"
+	BuildToolingRepoName                    = "eks-anywhere-build-tooling"
+	DefaultBaseRepoOwner                    = "aws"
+	BuildToolingRepoURL                     = "https://github.com/%s/eks-anywhere-build-tooling"
+	ReadmeFile                              = "README.md"
+	ReadmeUpdateScriptFile                  = "build/lib/readme_check.sh"
+	LicenseBoilerplateFile                  = "hack/boilerplate.yq.txt"
+	EKSDistroLatestReleasesFile             = "EKSD_LATEST_RELEASES"
+	EKSDistroProdReleaseNumberFileFormat    = "release/%s/production/RELEASE"
```

Comment:
where is this being used?

Reply:
This is used to read the prod release number for a particular Kubernetes release branch from the EKS Distro repo.